hironx_ros_bridge/rqt_plugin.xml

```diff
@@ -0,0 +1,22 @@
+<library path="src">
+  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
+    </description>
+    <qtgui>
+      <group>
+        <label>Robot</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to specific robots.</statustip>
+      </group>
+      <group>
```

Comment:
> how did rqt find the plugins? running external command?

rqt is depends on `pluginlib`. My guess about why I need `install` for rqt to find new plugins is because certain macros that `pluginlib` wants to see didn't get exported into `devel` space?